src/components/ScreenStackHeaderConfig.tsx

```diff
@@ -108,6 +115,6 @@ const styles = StyleSheet.create({
     width: '100%',
     flexDirection: 'row',
     justifyContent: 'space-between',
-    alignItems: 'center',
+    alignItems: Platform.OS === 'ios' ? 'center' : undefined,
```

Comment:
Yeah, I think we might be better off with this approach.

`minHeight` / `height` set to percents sets the value basing on the dimensions of the containing box in the css model **in the `ShadowTree`**. `AppBarLayout` does not exist in `ShadowTree`. `Screen` is the parent of the `ScreenStackHeaderConfig` in the `ShadowTree`.

Reply:
I think it'd be good to add a comment describing why this is done like this for people who edit this code in the future

Reply:
https://github.com/software-mansion/react-native-screens/pull/2456/commits/a6849f6a62729eda480300b014e135135d633254